spot-contracts/contracts/FeePolicy.sol

```diff
@@ -111,11 +113,11 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
     //-----------------------------------------------------------------------------
     // Incentive parameters
 
-    /// @notice The magnitude of perp's daily debasement (when dr <= 1).
-    uint256 public debasementValueChangePerc;
+    /// @notice The maximum allowed daily perp debasement percentage (when dr <= 1).
```

Comment:
Make these max values a percentage of combined TVL (SPOT + stAMPL), rather than percentage of just one side